dist/options/options.html

```diff
@@ -202,13 +202,16 @@ <h2 id="preferred-settings">Preferred Gmail settings</h2>
         <div class="tab">
             <h1>
                 Options
+            </h1>
+            <h2>
+                Label Filtering
```

Comment:
Maybe the heading can just be `Labels`? 

At some point later when there are more settings, it'd probably make sense to do away with the large headings